app/templates/main/index.html

```diff
@@ -16,10 +16,10 @@
         <div class="left-header">
           <div id="google_translate_element" class="translate"></div>
           <div id="links" class="hidden-xs">
-            <a href="http://www.asylumconnect.org/our-catalog/">ABOUT</a>
-            <a href="http://goo.gl/forms/UjxEbg6H3O">SHARE FEEDBACK</a>
-            <a href="{{ url_for('suggestion.suggest') }}">SUBMIT A RESOURCE</a>
-            <a href="http://www.asylumconnect.org/contact-us/">CONTACT US</a>
+            <a href="http://www.asylumconnect.org/our-cause/">ABOUT</a>
```

Comment:
actually the deleted links are the most up-to-date from https://github.com/AsylumConnect/asylumconnect.github.io/pull/37/files